assets/src/web-stories-block/block/transforms.js

```diff
@@ -135,6 +139,69 @@ const transforms = {
         },
       },
     },
+    {
+      type: 'block',
+      blocks: ['core/legacy-widget'],
+      isMatch: ({ idBase, instance }) => {
+        if (!instance?.raw) {
+          // Can't transform if raw instance is not shown in REST API.
+          return false;
+        }
+        return idBase === 'web_stories_widget';
+      },
+      transform: ({ instance }) => {
+        const {
+          raw: {
+            archive_link_label: archiveLinkLabel,
+            circle_size: circleSize,
+            image_alignment: imageAlignment,
+            number_of_columns: numOfColumns,
+            number_of_columns: numOfStories,
```

Comment:
```suggestion
            number_of_stories: numOfStories,
```

@spacedmonkey This should fix the issue reported in QA